atomic_reactor/utils/resource_manager.py

```diff
@@ -6,193 +6,290 @@
 of the BSD license. See the LICENSE file for details.
 """
 
-import os
-import logging
-import json
-import paramiko
 import backoff
+import datetime
+import json
 import jsonschema
-from typing import List, Any, Dict, Tuple
+import paramiko
+import random
+import os
 from datetime import datetime
 from shlex import quote
+from typing import Any, Dict, List, Tuple
+
+SSH_COMMAND_TIMEOUT = 120
+RELATIVE_PATH = 'resource_manager'
+RESOURCE_INFO_JSON = 'info.json'
+RETRY_ON_SSH_EXCEPTIONS = (paramiko.ssh_exception.AuthenticationException,
+                           paramiko.ssh_exception.ChannelException,
+                           paramiko.ssh_exception.NoValidConnectionsError,
+                           paramiko.ssh_exception.SSHException)
```

Comment:
```
Subclasses:     AuthenticationException, ChannelException, BadHostKeyException, IncompatiblePeer, ProxyCommandFailure, CouldNotCanonicalize, ConfigParseError
```

I don't think we have to explicitly mention `AuthenticationException` and `ChannelException`

Reply:
ok I'll remove those from list of exceptions